client/app/login/page.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import axios from "axios";
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+
+export default function Login() {
+  const router = useRouter();
+  const handleLoginRequest = () => {
+    console.log("Sending login request");
+
+    axios
+      .get("http://localhost:8000/auth/login/no-redirect", {})
+      .then(function (response) {
+        console.log(response.data.url);
+        router.push(response.data.url, { scroll: false });
+      })
+      .catch(() => {
+        console.log("Request failed");
+      });
+  };
+
+  return (
+    <main className="relative min-h-screen w-full h-full flex px-36">
+      <a href="/" className=" pt-4 w-full z-10">
+        <h1 className="text-3xl font-semibold text-[#1ED760]">Stagnum</h1>
+      </a>
+      <div className="top-0 left-0 w-full h-full absolute flex">
+        <Image
+          src={require("@/public/homeBG.jpg")}
+          fill
+          alt={"Home background"}
+        />
+      </div>
+      {/* <button onClick={handleLoginRequest}>Login button</button> */}
```

Comment:
We haven't yet written coding standards but I would suggest we avoid leaving commented out code in the codebase.

Reply:
Also having this commented out breaks the page. After I uncommented it I was able to visit the page.

Really dig the layout and design again😄 . I hope the pic is roalty-free because it fits perfectly (for light theme at least)